doc/getting_started/index.rst

```diff
@@ -358,17 +369,44 @@ each with their own way to install development tools:
              gnu-efi-devel \
              libuuid-devel \
              openssl-devel \
-             libpciaccess-devel
+             libpciaccess-devel \
+             systemd-devel \
+             libxml2-devel \
+             libevent-devel
 
-Build the hypervisor and device model
-=====================================
 
-#. Download the ACRN hypervisor and build it.
+Build the hypervisor, device model and tools
+============================================
+
+The `acrn-hypervisor <https://github.com/projectacrn/acrn-hypervisor/>`_
+repository has three main components in it:
+
+1. The ACRN hypervisor code located in the ``hypervisor`` directory
+#. The ACRN devicemodel code located in the ``devicemodel`` directory
+#. The ACRN tools source code located in the ``tools`` directory
 
+You can build all these components in one go as follows:
    .. code-block:: console
 
       $ git clone https://github.com/projectacrn/acrn-hypervisor
-      $ cd acrn-hypervisor/hypervisor
+      $ cd acrn-hypervisor
+      $ make
+
+   The build results are found in the ``build`` directory.
+
+   .. note::
+      if you wish to use a different target folder for the build
+      artefacts, set the ``BUILD_DIR`` to the desired value.
```

Comment:
include how you set the ``BUILD_DIR`` (with an example)

Reply:
An updated version is coming, with an example. The variable has changed too since then (see PR #260 ) and the new text reflects that.